lib/appmap/rspec.rb

```diff
@@ -5,6 +5,47 @@
 require 'appmap/inspect'
 require 'appmap/trace/tracer'
 
+if ActiveSupport::VERSION::MAJOR < 5
+  require 'active_support/inflector/transliterate'
+  require 'active_support/core_ext/string'
+  
+  module InflectorPatch
+    def parameterize(string, separator: "-", preserve_case: false, locale: nil)
```

Comment:
How does this differ from the existing behavior of `parameterize`? It is safe to monkey-patch it?